app.js

```diff
@@ -29,15 +29,18 @@ app.get('/beers', (req, res) => {
   .catch(error => console.log(error))
 })
 
-app.get('/beerCard', (req, res) => {
-  res.render('beerCard')
-})
-
 app.get('/random-beer', (req, res) => {
   punkAPI
   .getRandom()
   .then(responseFromApi => res.render('random-beer', {randomBeer: responseFromApi}))
   .catch(error => console.log(error))
 })
 
+app.get('/beers/:id', (req, res) => {
+  punkAPI
+  .getBeer(req.params.id)
+  .then(data => res.render('beerDetails', {clickedBeer: data}))
+  .catch(error => console.log(error))
+})
+
```

Comment:
Great job Xiaomei! Keep up the good work! 👍🏽

Reply:
Thanks Kevin 👍